tests/tags.yml

```diff
@@ -0,0 +1,5 @@
+pipeline_nf-aggregate:
```

Comment:
Can we put this file in the `workflows/nf_aggregate/` folder instead so everything is self-enclosed.

Reply:
Ok. I'll move as required. Thanks!